### ISSUE TITLE
Wrong MSE computation in the SSIM code example

### ISSUE THREAD

**Issue by TY48** [scikit-image/scikit-image#4743]
## Description

In the code example of SSIM (https://scikit-image.org/docs/dev/auto_examples/transform/plot_ssim.html), computation of mse (Mean Squared Error) seems incorrect. 

Currently it is 
```python
def mse(x, y):
    return np.linalg.norm(x - y)
```
If I'm not wrong it should be something like this 
```python
def mse(x, y):
    return ((x-y)**2).mean()
```

## Version information
Docs for 0.18.dev0

**Comment by rfezzani** [comment]
Thank you @TY48 for the report, this is so weird!
BTW, I think that if adapted, the use of `np.linalg.norm` still have some performance advantages:
```python
In [1]: img0 = np.random.rand(1000, 1000)                                                                              

In [2]: img1 = np.random.rand(1000, 1000)                                                                              

In [3]: ((img1 - img0) ** 2).mean()                                                                                    
Out[3]: 0.16620402086323402

In [4]: np.linalg.norm(img1 - img0) ** 2 / img1.size                                                                   
Out[4]: 0.16620402086323394

In [5]: %timeit ((img1 - img0) ** 2).mean()                                                                            
3.62 ms ± 16.7 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)

In [6]: %timeit np.linalg.norm(img1 - img0) ** 2 / img1.size                                                           
2.59 ms ± 19.9 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
```

**Comment by grlee77** [comment]
Yeah, the example named the variable MSE but it is actually computing RMSE (root mean square error). We actually have a simple mse function that has the form you were expecting:
https://github.com/scikit-image/scikit-image/blob/23829977f23c957dbff24757069dbdc4d929248f/skimage/metrics/simple_metrics.py#L44

I would just use that mean_square_error function instead of defining it in the example.

**Comment by TY48** [comment]
Thank you @rfezzani  and @grlee77 

Simply using the existing mean_square_error function instead of redefining in the example sounds good.

Just to be precise, L2 norm (current implementation) corresponds to the square root of the sum of the squared vector values (not RMSE).

**Comment by jjerphan** [comment]
This issue can be closed. :slightly_smiling_face:

**Comment by grlee77** [comment]
closed by #4807
